test/bdd/fixtures/edv-rest/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+services:
+
+  edv.example.com:
+    container_name: edv.example.com
+    image: ${EDV_REST_IMAGE}:${EDV_REST_IMAGE_TAG}
+    environment:
+      - EDV_HOST_URL=${EDV_HOST}:${EDV_PORT}
+    ports:
+      - ${EDV_PORT}:${EDV_PORT}
+    command: start
+    networks:
+      - bdd_net
```

Comment:
Just curious: why we need network here since I can see only one docker-compose here?

Reply:
You're right; I don't need it - it's removed now.